packages/extension-ui/src/components/Address.tsx

```diff
@@ -504,9 +506,13 @@ export default styled(Address)(
   }
 
   .fullAddress {
+    display: flex;
+    align-items: center;
```

Comment:
Not better `space-between`? Wont' `center` cut away too much of the the address?

Reply:
I center them vertically -- it shouldn't impact available width

Reply:
Of course, my bad 🤦